src/server/req_message.c

```diff
@@ -271,10 +242,10 @@ void
 req_relnodesjob(struct batch_request *preq)
 {
 	int             jt;		/* job type */
+	int             js;     /* job state */
```

Comment:
Format this comment so as to align with the previous line.

Reply:
Fixed in rc 17 https://github.com/PBSPro/pbspro/pull/590/commits/f94a3405ef13edaa87df40a89af36420b9627c7b